spectral_cube/stokes_spectral_cube.py

```diff
@@ -106,6 +122,19 @@ def __dir__(self):
     def components(self):
         return list(self._stokes_data.keys())
 
+    @property
+    def stokes_type(self):
+        """
+        What kind of stokes has been setup.
+        It can sky, linear, circular, generic or other.
+        Where the sky refers to the the stokes in sky basis.
+        Linear to the stokes in linear feed basis.
+        Circular to stokes in circularl feed basis.
+        Generic to the a general four orthagonal components.
+        Other refers to the non standard stokes types.
```

Comment:
is this a poem?  Kind looks like one!  Could you clean up the prose if not?  

note typo in `circular`

Reply:
Fixed the text and the typo thanks.